### PR TITLE
Alert simulator security microsoft.security 2021 01 01

### PR DESCRIPTION
**Pull request by yuvi123yuvi**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Changelog
Please ensure to add changelog with this PR by answering the following questions.
  1. What's the purpose of the update?    
      - [ ] new service onboarding 
      - [x] new API version 
      - [ ] update existing version for new feature 
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify 
  2. When you are targeting to deploy new service/feature to public regions? Please provide date, or month to public if date is not available yet.
  3. When you expect to publish swagger? Please provide date, or month to public if date is not available yet.
  4. If it's an update to existing version,  please select SDKs of specific language and CLIs that require refresh after swagger is published.
      - [x] SDK of .NET (need service team to ensure code readiness)
      - [x] SDK of Python
      - [x] SDK of Java
      - [x] SDK of Js
      - [x] SDK of Go
      - [x] PowerShell
      - [x] CLI
      - [x] Terraform
      - [ ] No, no need to refresh for updates in this PR

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] Please ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

File: specification/security/resource-manager/Microsoft.Security/stable/2021-01-01/alerts.json [hunk 1]
```diff
@@ -0,0 +1,1107 @@
...
+            ]
+          }
+        }
+      }
+    },
+    "AlertSimulatorRequestBody": {
+      "type": "object",
+      "description": "Alert Simulator request body.",
+      "properties": {
+        "properties": {
```
Comment (pilor) [inline specification/security/resource-manager/Microsoft.Security/stable/2021-01-01/alerts.json:998]: POST request bodies don't need to have their properties inside a "properties" property.  You can keep it this way or move them to the top level, either is fine.

**Review by pilor — APPROVED** [review]

File: specification/security/resource-manager/Microsoft.Security/stable/2021-01-01/alerts.json [hunk 2]
```diff
@@ -559,8 +559,8 @@
           }
         ],
         "responses": {
-          "204": {
-            "description": "No Content"
+          "202": {
```
Comment (pilor) [inline specification/security/resource-manager/Microsoft.Security/stable/2021-01-01/alerts.json:562]: this needs to be marked x-ms-long-running-operation if it returns 202

**Comment by ChenTanyi** [comment]
@yuvi123yuvi please fix the duplicate schema in SDK validation.

File: specification/security/resource-manager/Microsoft.Security/stable/2021-01-01/alerts.json (hunk 2 above)
Comment (yuvi123yuvi) [reply specification/security/resource-manager/Microsoft.Security/stable/2021-01-01/alerts.json:566]: Added :)

File: specification/security/resource-manager/Microsoft.Security/stable/2021-01-01/alerts.json (hunk 1 above)
Comment (yuvi123yuvi) [reply specification/security/resource-manager/Microsoft.Security/stable/2021-01-01/alerts.json:1002]: Ok thanks, keeping it this way.

**Comment by ChenTanyi** [comment]
As @yuvi123yuvi said, the validation seems false positive. Then, the PR is OK to merge. @yuvi123yuvi Please let me know when it ready to merge.

**Comment by changlong-liu** [comment]
Hi @yuvi123yuvi , 
we find  ["final-state-via": "original-uri"] is used in operation "Alerts_Simulate": https://github.com/Azure/azure-rest-api-specs/blob/main/specification/security/resource-manager/Microsoft.Security/stable/2021-01-01/alerts.json#L563

With this setting, client side will try to visit this url with GET after the POST "Alerts_Simulate" operation. But actually there is no GET defined for this url. So want to know is there a GET interface in service side for this operation? Otherwise ["final-state-via": "original-uri"] should don't be defined in the "Alerts_Simulate".

CC to @pilor .

**Pull request merged**